x64 Indexer packages failed to be built at the `build-wazuh-plugins` stage

## Description

While analyzing https://github.com/wazuh/devel-major-incidents/issues/104, it has been detected that the x64 Indexer packages were not generated due to the following error at the `build-wazuh-plugins` stage:

```
> Task :integTest FAILED
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/__run-1778204679562.json

51 actionable tasks: 51 executed
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':integTest'.
> There were failing tests. See the report at: file:///home/runner/work/wazuh-indexer/wazuh-indexer/plugins/content-manager/build/reports/tests/integTest/index.html
```

Further investigation is required.

## Evidence

- https://github.com/wazuh/wazuh-indexer/actions/runs/25530860925/job/74939842212
- https://github.com/wazuh/wazuh-indexer/actions/runs/25530814809/job/74939842718

--- a/plugins/content-manager/build.gradle
+++ b/plugins/content-manager/build.gradle
@@ -207,6 +207,14 @@ run {
   useCluster testClusters.integTest
 }
 
+// When this build is rooted at content-manager, an unqualified `./gradlew run`
+// would also fire the included :wazuh-indexer-setup project's own `run` task,
+// which boots a separate cluster on the same default port and shadows ours.
+// Disable it so only this project's `run` is invoked.
+project(':wazuh-indexer-setup').afterEvaluate { setupProject ->
+  setupProject.tasks.named('run').configure { enabled = false }
+}
+
 // updateVersion: Task to auto update version to the next development iteration
 task updateVersion {
   onlyIf { System.getProperty('newVersion') }

--- a/plugins/content-manager/build.gradle
+++ b/plugins/content-manager/build.gradle
@@ -160,7 +160,7 @@ testClusters.integTest {
   // Enable memory tracking to diagnose OOM errors.
   jvmArgs '-Xlog:gc*:file=gc.log:time,uptime,level,tags:filecount=5,filesize=10m'
   jvmArgs '-XX:NativeMemoryTracking=summary'
-  jvmArgs '-Xms1g', '-Xmx1g'
+  jvmArgs '-Xms2g', '-Xmx2g'
   jvmArgs '-Xss2m'
 
   // The plugin's REST handlers use blocking actionGet() calls on transport threads.

--- a/plugins/content-manager/src/main/java/com/wazuh/contentmanager/cti/catalog/service/ConsumerRulesetService.java
+++ b/plugins/content-manager/src/main/java/com/wazuh/contentmanager/cti/catalog/service/ConsumerRulesetService.java
@@ -153,7 +153,6 @@ public class ConsumerRulesetService extends AbstractConsumerService {
                     Constants.INDEX_DECODERS,
                     Constants.INDEX_KVDBS,
                     Constants.INDEX_INTEGRATIONS,
-                    Constants.INDEX_FILTERS,
                     Constants.INDEX_POLICIES);
 
             // Sync Integrations

--- a/plugins/setup/src/main/resources/templates/content/filters.json
+++ b/plugins/setup/src/main/resources/templates/content/filters.json
@@ -50,30 +50,30 @@
             "metadata": {
               "properties": {
                 "author": {
-                  "properties": {
-                    "date": {
-                      "type": "date"
-                    },
-                    "email": {
-                      "ignore_above": 1024,
-                      "type": "keyword"
-                    },
-                    "modified": {
-                      "type": "date"
-                    },
-                    "name": {
-                      "ignore_above": 1024,
-                      "type": "keyword"
-                    },
-                    "url": {
-                      "ignore_above": 1024,
-                      "type": "keyword"
-                    }
-                  }
+                  "ignore_above": 1024,
+                  "type": "keyword"
+                },
+                "date": {
+                  "type": "date"
                 },
                 "description": {
                   "norms": false,
                   "type": "text"
+                },
+                "documentation": {
+                  "ignore_above": 1024,
+                  "type": "keyword"
+                },
+                "modified": {
+                  "type": "date"
+                },
+                "references": {
+                  "ignore_above": 1024,
+                  "type": "keyword"
+                },
+                "supports": {
+                  "ignore_above": 1024,
+                  "type": "keyword"
                 },
                 "title": {
                   "ignore_above": 1024,

--- a/plugins/setup/src/main/resources/templates/content/filters.json
+++ b/plugins/setup/src/main/resources/templates/content/filters.json
@@ -74,6 +74,10 @@
                 "description": {
                   "norms": false,
                   "type": "text"
+                },
+                "title": {
+                  "ignore_above": 1024,
+                  "type": "keyword"
                 }
               }
             },

--- a/wcs/content/filters/docs/fields.csv
+++ b/wcs/content/filters/docs/fields.csv
@@ -4,12 +4,13 @@ ECS_Version,Indexed,Field_Set,Field,Type,Level,Normalization,Example,Description
 9.1.0,true,document,document.check,object,custom,,event.action:login AND event.outcome:failure,Check expression or identifier used by the filter.
 9.1.0,true,document,document.enabled,boolean,custom,,True,Whether the filter is enabled.
 9.1.0,true,document,document.id,keyword,custom,,filter-12345,Unique identifier of the filter document.
-9.1.0,true,document,document.metadata.author.date,date,custom,,2024-01-15T08:30:00.000Z,Creation date of the filter metadata.
-9.1.0,true,document,document.metadata.author.email,keyword,custom,,security@example.com,Email of the filter author.
-9.1.0,true,document,document.metadata.author.modified,date,custom,,2024-02-20T14:45:00.000Z,Last modification date of the filter metadata.
-9.1.0,true,document,document.metadata.author.name,keyword,custom,,SecOps Team,Name of the filter author.
-9.1.0,true,document,document.metadata.author.url,keyword,custom,,https://example.com/filters,URL associated with the filter author or documentation.
+9.1.0,true,document,document.metadata.author,keyword,custom,,"Wazuh, Inc.",Author of the filter.
+9.1.0,true,document,document.metadata.date,date,custom,,2024-01-15T08:30:00.000Z,Creation date of the filter.
 9.1.0,true,document,document.metadata.description,text,custom,,Detects failed logins from new locations.,Description of the filter purpose or behavior.
+9.1.0,true,document,document.metadata.documentation,keyword,custom,,https://documentation.wazuh.com/current/user-manual/ruleset/filters.html,Documentation link for the filter.
+9.1.0,true,document,document.metadata.modified,date,custom,,2024-02-20T14:45:00.000Z,Last modification date of the filter.
+9.1.0,true,document,document.metadata.references,keyword,custom,,https://documentation.wazuh.com/current/user-manual/ruleset/filters.html,References or links related to the filter.
+9.1.0,true,document,document.metadata.supports,keyword,custom,,linux,Supported platforms or components for the filter.
 9.1.0,true,document,document.metadata.title,keyword,custom,,Suspicious Login Filter,Title of the filter.
 9.1.0,true,document,document.name,keyword,custom,,Block suspicious login,Human-readable name of the filter.
 9.1.0,true,document,document.type,keyword,custom,,event,Filter type or category.

--- a/wcs/content/filters/docs/fields.csv
+++ b/wcs/content/filters/docs/fields.csv
@@ -10,6 +10,7 @@ ECS_Version,Indexed,Field_Set,Field,Type,Level,Normalization,Example,Description
 9.1.0,true,document,document.metadata.author.name,keyword,custom,,SecOps Team,Name of the filter author.
 9.1.0,true,document,document.metadata.author.url,keyword,custom,,https://example.com/filters,URL associated with the filter author or documentation.
 9.1.0,true,document,document.metadata.description,text,custom,,Detects failed logins from new locations.,Description of the filter purpose or behavior.
+9.1.0,true,document,document.metadata.title,keyword,custom,,Suspicious Login Filter,Title of the filter.
 9.1.0,true,document,document.name,keyword,custom,,Block suspicious login,Human-readable name of the filter.
 9.1.0,true,document,document.type,keyword,custom,,event,Filter type or category.
 9.1.0,true,hash,hash.sha256,keyword,custom,,e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855,The SHA-256 hash of the filter document.

--- a/wcs/content/filters/fields/custom/filter.yml
+++ b/wcs/content/filters/fields/custom/filter.yml
@@ -29,6 +29,12 @@
       description: >
           Whether the filter is enabled.
       example: true
+    - name: document.metadata.title
+      type: keyword
+      level: custom
+      description: >
+          Title of the filter.
+      example: "Suspicious Login Filter"
     - name: document.metadata.description
       type: text
       level: custom

--- a/wcs/content/filters/fields/custom/filter.yml
+++ b/wcs/content/filters/fields/custom/filter.yml
@@ -41,36 +41,42 @@
       description: >
           Description of the filter purpose or behavior.
       example: "Detects failed logins from new locations."
-    - name: document.metadata.author.email
+    - name: document.metadata.author
       type: keyword
       level: custom
       description: >
-          Email of the filter author.
-      example: "security@example.com"
-    - name: document.metadata.author.name
-      type: keyword
-      level: custom
-      description: >
-          Name of the filter author.
-      example: "SecOps Team"
-    - name: document.metadata.author.url
-      type: keyword
-      level: custom
-      description: >
-          URL associated with the filter author or documentation.
-      example: "https://example.com/filters"
-    - name: document.metadata.author.date
+          Author of the filter.
+      example: "Wazuh, Inc."
+    - name: document.metadata.date
       type: date
       level: custom
       description: >
-          Creation date of the filter metadata.
+          Creation date of the filter.
       example: "2024-01-15T08:30:00.000Z"
-    - name: document.metadata.author.modified
+    - name: document.metadata.modified
       type: date
       level: custom
       description: >
-          Last modification date of the filter metadata.
+          Last modification date of the filter.
       example: "2024-02-20T14:45:00.000Z"
+    - name: document.metadata.references
+      type: keyword
+      level: custom
+      description: >
+          References or links related to the filter.
+      example: "https://documentation.wazuh.com/current/user-manual/ruleset/filters.html"
+    - name: document.metadata.documentation
+      type: keyword
+      level: custom
+      description: >
+          Documentation link for the filter.
+      example: "https://documentation.wazuh.com/current/user-manual/ruleset/filters.html"
+    - name: document.metadata.supports
+      type: keyword
+      level: custom
+      description: >
+          Supported platforms or components for the filter.
+      example: "linux"
     - name: document.check
       type: object
       enabled: false

